demo/blocks_templates/run.py

```diff
@@ -0,0 +1,11 @@
+import gradio as gr
+user_db = {"admin": "admin", "foo": "bar"}
+
+
+def greet(name):
+    return "Hello " + name + "!!"
+
+
+demo = gr.Interface(fn=greet, inputs=gr.Templates.Textbox(), outputs=gr.Templates.Textbox())
```

Comment:
I suggest using a more interesting template, such as `Webcam` for this demo.

Also, I don't see a template called `Textbox`

Reply:
My bad, it was going to be HugeTextbox, will update accordingly.